src/components/NavBar/NavBar.jsx

```diff
@@ -57,6 +94,17 @@ NavBar.propTypes = {
       onItemClick: PropTypes.func,
     })).isRequired,
   }).isRequired,
+  helpMenuItems: PropTypes.arrayOf(PropTypes.shape({
+    id: PropTypes.string.isRequired,
+    title: PropTypes.string.isRequired,
+    component: PropTypes.node,
+    hasDivider: PropTypes.bool,
+    onItemClick: PropTypes.func,
```

Comment:
a super small one Hamish, do you think we can change this to `onClick`😄?

Reply:
Thanks Fed! This is actually just what I pass as the items to the `<Select />` component, so [it needs to be `onItemClick`](https://github.com/bufferapp/ui/blob/0bcf339f1b0a5ad3069d84708b402c61ad41a0f2/src/components/Select/Select.jsx#L254), though I do think we could have that changed.

Reply:
Yeah saw that. sorry should have been more clear, was thinking if we could change it in the `select`. I just noticed this for the first time in this PR 😄. 

Again not a big one, just stood out to me when I was looking at the code.